src/Components/AdministratorPage/CycleJobCategories/CycleJobCategories.jsx

```diff
@@ -2,69 +2,73 @@ import { useEffect, useState } from 'react';
 import { withRouter } from 'react-router';
 import { Link } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
-import Picky from 'react-picky';
 import FA from 'react-fontawesome';
 import swal from '@sweetalert/with-react';
 import Spinner from 'Components/Spinner';
 import TabbedCard from '../../TabbedCard/TabbedCard';
 import CheckBox from '../../CheckBox/CheckBox';
-import { cycleJobCategoriesData, cycleJobCategoriesEdit, cycleJobCategoriesFilters } from '../../../actions/cycleJobCategories';
-import ListItem from '../../BidderPortfolio/BidControls/BidCyclePicker/ListItem/ListItem';
+import { cycleCategories as cycleCategoriesList, cycleJobCategories, cycleJobCategoriesEdit, cycleJobCategoriesStatuses } from '../../../actions/cycleJobCategories';
```

Comment:
```suggestion
import { cycleCategories as cycleCategoriesList, cycleJobCategories, cycleJobCategoriesEdit, cycleJobCategoriesStatuses } from 'actions/cycleJobCategories';
```
have to move it up to come before the relative paths after making absolute